app/src/org/commcare/activities/EntitySelectActivity.java

```diff
@@ -308,64 +339,39 @@ private void restoreExistingSelection(boolean isOrientationChange) {
         }
     }
 
-    private void setupToolbar(AdapterView view) {

```

Comment:
I also confirmed on a pre-honeycomb device that the footer still shows up properly